extensions/sql-database-projects/src/common/apiWrapper.ts

```diff
@@ -115,10 +115,18 @@ export class ApiWrapper {
 		return vscode.window.showQuickPick(items, options, token);
 	}
 
+	public showQuickPickString(items: string[] | Thenable<string[]>, options?: vscode.QuickPickOptions, token?: vscode.CancellationToken): Thenable<string | undefined> {
+		return vscode.window.showQuickPick(items, options, token);
+	}
+
```

Comment:
Any reason to go not go with the QuickPickItem version that's already in the wrapper?

Reply:
I was just trying to take the easy route. But updated it now.